pallets/did/src/origin.rs

```diff
@@ -50,9 +50,12 @@ where
 
 	#[cfg(feature = "runtime-benchmarks")]
 	fn successful_origin() -> OuterOrigin {
+		let zero_account_id = AccountId::decode(&mut sp_runtime::traits::TrailingZeroInput::zeroes())
+			.expect("infinite length input; no invalid inputs for type; qed");
+
 		OuterOrigin::from(DidRawOrigin {
-			id: Default::default(),
-			submitter: Default::default(),
+			id: zero_account_id.clone().into(),
+			submitter: zero_account_id,
```

Comment:
This might be an incorrect solution. In any case, we should not require `Default` for `AccountId` types as it is not implemented anymore due to the DefaultAccount "issue".